.travis.yml

```diff
@@ -5,6 +5,7 @@ language: php
 cache:
     directories:
         - $HOME/.composer/cache/files
+        - .phpunit
```

Comment:
The bridge will not use this directory by default, will it? IIRC we need to use the `PHPUNIT_DIR` env var to point to this directory.

Reply:
done